packages/compat/src/components/Loader/index.tsx

```diff
@@ -1,12 +1,14 @@
 'use client';
 
+import type { CommonProps } from '@ui5/webcomponents-react';
 import { deprecationNotice, useI18nBundle, useStylesheet } from '@ui5/webcomponents-react-base';
 import { clsx } from 'clsx';
 import type { CSSProperties } from 'react';
 import { forwardRef, useEffect, useState } from 'react';
-import { LoaderType } from '../../enums/index.js';
-import { PLEASE_WAIT } from '../../i18n/i18n-defaults.js';
-import type { CommonProps } from '../../types/index.js';
+// todo: sb won't start with this (maybe because of vite.config alias paths?)
+// import { PLEASE_WAIT } from '@ui5/webcomponents-react/dist/i18n/i18n-defaults.js';
+import { PLEASE_WAIT } from '../../../../main/src/i18n/i18n-defaults.js';
```

Comment:
sounds like a vite config issue to be honest. Either we have to expose the i18n defaults in our main export, or add the config